Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml

```diff
@@ -0,0 +1,76 @@
+# Uses the chessboard and generic pieces
+- type: entity
+  parent: BaseItem
+  id: CheckerBoard
+  name: checkerboard
+  description: A checkerboard. Pieces included!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard.rsi
+    state: chessboard
+  - type: TabletopGame
+    boardName: tabletop-checker-board-name
+    size: 338, 274
+    setup: !type:TabletopCheckerSetup
+
+# Chessboard tabletop item (item only visible in tabletop game)
+- type: entity
+  id: CheckerBoardTabletop
+  name: checkerboard
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard_tabletop.rsi
+    state: chessboard_tabletop
```

Comment:
would be nice if it used a unique sprite but i suppose checkerboards are sometimes the same as chessboards